app/models/spree/payment_method/sisow_billing/paypalec.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  class PaymentMethod::SisowBilling
+    class Paypalec < SisowPaymentMethod
+      def redirect_url(order, opts = {})
+        sisow = PaymentMethod::SisowBilling.new(order)
+        sisow.start_transaction('paypalec', opts)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.